### PR TITLE
powersink power needed to 100 MW from 10 GW, drain from 1.6 MW to 1 MW

### PR DESCRIPTION
**Pull request by Commandersand**
## Why It's Good For The Game
they never blow up, 

## Changelog
:cl:
balance: powersinks need a lot less power to blow up now and drain 1MW  down from 1.6
/:cl:

### CODE REVIEW

**Comment by LimblessLayabout** [comment]
Was it ever the intention that they act as a bomb? I was of the impression this was more a condition to defeat a powersink through engineering - simply crank the engine output til the damn thing detonates, rather than try to track down and destroy the object in person. The power cap is arbitrarily high so it doesn't just.. _eventually_ expire naturally under normal conditions.

File: code/game/objects/items/devices/powersink.dm [hunk 1]
```diff
@@ -18,7 +18,7 @@ GLOBAL_LIST_EMPTY(power_sinks)
 	custom_materials = list(/datum/material/iron=750)
 	var/drain_rate = 1600000	// amount of power to drain per tick
 	var/power_drained = 0 		// has drained this much power
-	var/max_power = 1e10		// maximum power that can be drained before exploding
+	var/max_power = 1e8		// maximum power that can be drained before exploding && SKYRATE CHANGE form 1e10
```
Comment (CinderWC) [inline code/game/objects/items/devices/powersink.dm:21]: Spell Check

**Comment by CinderWC** [comment]
I dont know about actually **buffing** powersinks when people want to have the objective for it removed.

Also the fact that this buff was made without any counterbalance...

**Comment by Commandersand** [comment]
i dunno, it's meant to explode eventually but it never happens because of how high the energy you need. 
Also it's now rebalanced to drain 1 MW so if you have a good engine setup you'll still have some power in

**Comment by IronEleven** [comment]
I think the explosion serves as a better consequence for failing to correctly handle the power sink than no power for the rest of the round.

**Comment by Useroth** [comment]
Back in the old days, you would hotwire a singulo into the grid to make sinks explode. Gonna leave this one up for more discussion

**Pull request merged**